nautobot/ipam/templates/ipam/ipaddress_merge.html

```diff
@@ -0,0 +1,237 @@
+                                        {% elif value.has_many and value.queryset.count %}
+                                            <td>
+                                                <input type="radio" id="cr_{{relationship.key}}" name="cr_{{relationship.key}}" value="{{ value.queryset | queryset_to_pks }}" required {% if forloop.parentloop.parentloop.first %} checked {% endif %}>
+                                                <label for="cr_{{relationship.key}}"> 
+                                                    <a href="{% url 'extras:relationshipassociation_list' %}?relationship={{relationship.key}}&{{side}}_id={{object.id}}">
+                                                        {{ value.queryset.count }}
+                                                        {% if value.peer_type.model_class is not None %}
+                                                            {% if value.queryset.count > 1 %}
+                                                                {{ value.peer_type.model_class|meta:"verbose_name_plural" }}
+                                                            {% else %}
+                                                                {{ value.peer_type.model_class|meta:"verbose_name" }}
+                                                            {% endif %}
+                                                        {% else %}
+                                                            {{ value.peer_type }}(s)
+                                                        {% endif %}
+                                                    </a>
+                                            </label>
+                                            </td>
+                                        {% else %}
+                                            <td>
+                                                <input type="radio" id="cr_{{relationship.key}}" name="cr_{{relationship.key}}" value="" required {% if forloop.parentloop.parentloop.first %} checked {% endif %}>
+                                                <label for="cr_{{relationship.key}}"><span class="text-muted">&mdash;</span></label>
+                                            </td>
+                                        {% endif %}
+                                    {% endfor %}
+                                {% endfor %}
+                            {% endif %}
+                        {% endwith %}
+                        </tr>
+                {% endfor %}
+                </tbody>    
+            </table>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-5 col-md-offset-7">
+        {% block buttons %}
+            <button type="submit" name="_merge" class="btn btn-primary"><i class="mdi mdi-arrow-down-bold"></i> Merge and Go to the Next Duplicate</button>
+            <button type="submit" name="_skip" class="btn btn-warning"><i class="mdi mdi-arrow-right-bold"></i> Skip and Go to the Next Duplicate</button>
```

Comment:
Possibly `mdi-merge`/`mdi-set-merge` for the merge button and/or `mdi-debug-step-over` for the skip button?